config/orchestrations/landing/0.1/kabanero-landing.yaml

```diff
@@ -19,7 +21,7 @@ spec:
     kind: Service
     name: kabanero-landing
   tls:
-    termination: passthrough
+    termination: reencrypt
```

Comment:
Since this also requires changes to the landing page container, I think we need to make a `0.2` orchestration with this change.  Otherwise, if someone uses kabanero-operator 0.8.0 and codes `spec.version` to something like `0.7.0`, the landing page won't work.

Reply:
Thanks Tim, Yeah that is very true. A bit more complicated issue is if someone deploys a CR instance with spec.version = 0.8.0 and decides to use landing version 0.7.0. In this case, the landing page won't come up, and it becomes a bit harder to seamlessly toggle the kabanero updates because we now need to introduce a restriction on the landing page version that can be used.

Reply:
I understand - I consider that an edge case.  I think we need to support setting `spec.version` and expecting that the customer would take all of the rest of the defaults.  If we also support `spec.landing.version` then we can support the case where the customer has `spec.version` set to `0.8.0` but wants to use a prior version of the landing page for some reason.  The code that deploys the landing page would use the right version of the orchestration based on the value of `spec.landing.version` since the orchestration is coded into the version struct.  There would need to be special case code in `landing.go` that behaved differently based on the orchestration version.